UPGRADE-3.x.md

```diff
@@ -40,7 +40,7 @@ You MUST use Symfony's [`help`](https://symfony.com/doc/4.4/reference/forms/type
 Before:
 ```php
 $formMapper
-    ->add('field', null, [
+    ->add('field', null, [], [
```

Comment:
Good catch! can we add both examples?

Reply:
setHelp & setHelps ?

Reply:
Done

Reply:
Well I meant like:
```php
->add('field_with_help_in_form_options', null, ['help' => '...'])
->add('field_with_help_in_field_description_options', null, [], ['help' => '...'])
```

Reply:
Maybe.. But in new example we showing two moments how stop throwing deprecating notices